### ISSUE TITLE
search by imdb_id

### ISSUE THREAD

**Issue by tanto-grs** [CIAvash/WebService-TMDB#1]
Dear CIAvash,
i find the WebService-TMDB module for Raku very nice and i think this it not an issue, but how can i search a movie by the IMDB-ID? I look everything i can found in this module but i didn't find a way.

can u help me to solve the issue?

**Comment by CIAvash** [comment]
It seems I have not implemented that part, which is [`find/external_id`](https://developer.themoviedb.org/reference/find-by-id).

I'll take a look when I get the time, but if your're in a hurry, you can use the `request` method of the `WebService::TMDB` class to make the request you want. An (untested) example:
```raku
$tmdb.request.get: "find/$imdb_id", params => %(external_source => 'imdb_id')
```

**Comment by tanto-grs** [comment]
Dear CIAvash,
thx for the fast answer, i will try the snippet you wrote and i hope you will find time to implement that part.

Best regards

**Comment by CIAvash** [comment]
The issue is resolved in version 0.1.2.

For examples, you can look at [README](https://codeberg.org/CIAvash/WebService-TMDB) and [tests](https://codeberg.org/CIAvash/WebService-TMDB/src/commit/54ae291fadff845bf008cb3292551de5be110429/t/99-TMDB.rakutest#L17)
